docs/04-operation-guides/operations/evnt-03-nats-server-system-events.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Acquiring NATS server system account credentials
+---
+## Context
+
+Accessing certain resources in NATS requires [`system_account` privileges](https://docs.nats.io/running-a-nats-service/configuration/sys_accounts). Kyma will automatically generate a `system account` user via a Secret dubbed `eventing-nats-secret` in the `kyma-system` Namespace.
+
+## Prerequisites
+
+None.
```

Comment:
I guess users need to have a certain Kyma version installed? Or will this work in all versions?